package.json

```diff
@@ -143,6 +143,7 @@
     "@electron/fuses": "1.6.0",
     "@electron/universal": "1.3.1",
     "@mattermost/compass-icons": "0.1.32",
+    "@mattermost/desktop-api": "*",
```

Comment:
We actually do this same thing [here](https://github.com/mattermost/mattermost/blob/62a359299ac4022e5efbf5f8837977cdac331605/webapp/channels/package.json#L18) for the webapp, I believe we do that to make sure we use the local package instead of the remote one while developing.

We want the web app to depend on a specific version so we know how the compatibility works, but for the Desktop App it should always use whatever types are available within the same commit as the app was built.